### PR TITLE
Bugfix/remove length restriction on product name

### PR DESCRIPTION
**Pull request by simone-van-zyl**

### CODE REVIEW

**Review by HarryGHeathcock — COMMENTED** [review]
Minor version bump makes sense, I'm unsure on how patch versions work so I don't know why it needs to jump to RC3.

File: src/main/java/io/electrum/airtime/api/model/Product.java [hunk 1]
```diff
@@ -143,7 +143,7 @@ public Product name(String name) {
     **/
    @ApiModelProperty(value = "The name of the product.")
    @JsonProperty("name")
-   @Length(min = 1, max = 40)
+   @Length(min = 1, max = 80)
```
Comment (HarryGHeathcock) [inline src/main/java/io/electrum/airtime/api/model/Product.java:146]: Change to 120

File: src/docs/devguide/hugo/site/content/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 This page describes changes to the Airtime Service Interface implemented across different releases of the interface.
 
+## v5.24.0
+Release 09 April 2021
```
Comment (HarryGHeathcock) [inline src/docs/devguide/hugo/site/content/release-notes.md:4]: Should this be 23 April 2021?

**Review by HarryGHeathcock — COMMENTED** [review]
Change is implemented properly.

File: src/docs/devguide/hugo/site/content/release-notes.md (hunk 2 above)
Comment (simone-van-zyl) [reply src/docs/devguide/hugo/site/content/release-notes.md:4]: Nice catch!

**Review by HarryGHeathcock — COMMENTED** [review]
Changes and dates are all good

**Review by HarryGHeathcock — APPROVED** [review]
Consider this approved

**Pull request merged**